### PR TITLE
feat(stream): add limit-conn

### PR DESCRIPTION
**Pull request by spacewander**
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/zh/latest/plugins/limit-conn.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ title: limit-conn
 
 **注：key 是可以被用户自定义的，只需要修改插件的一行代码即可完成。并没有在插件中放开是处于安全的考虑。**
 
+: when used in the stream proxy, only `remote_addr` and `server_addr` can be used as `key`. And `rejected_code` is meaningless.
```
Comment (tokers) [inline docs/zh/latest/plugins/limit-conn.md:38]: What is the meaning of the leading colon symbol?

**Review by tzssangglass — APPROVED** [review]

**Review by tokers — APPROVED** [review]

File: t/stream-plugin/limit-conn.t [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                ngx.HTTP_PUT,
+                [[{
+                    "nodes": {
+                        "127.0.0.1:1995": 1
+                    },
+                    "type": "roundrobin"
+                }]]
+                )
+
+            if code >= 300 then
```
Comment (tao12345666333) [inline t/stream-plugin/limit-conn.t:100]: Should it only be restricted to not equal to 200 or 201? Although currently its value will only be 200, 201 or 500

**Review by tao12345666333 — APPROVED** [review]
Approved.
Leave a comment, but it doesn’t affect.

File: t/stream-plugin/limit-conn.t (hunk 2 above)
Comment (spacewander) [reply t/stream-plugin/limit-conn.t:100]: `code >= 300` acts as `err != nil`, since it is so widely used, we can keep using it.

**Pull request merged**